CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+## Version 0.1.5
+
+[diff since 0.1.4](https://github.com/QEDjl-project/QEDbase.jl/compare/0c70f66...release-0.1.5)
+
+### Breaking changes
+
+no breaking changes
+
+### New features
+
+* Move particle definitions from QEDprocesses.jl to QEDbase.jl https://github.com/QEDjl-project/QEDbase.jl/pull/25
+* Base state fix https://github.com/QEDjl-project/QEDbase.jl/pull/37
+
+### Maintenance
+
+* CompatHelper: bump compat for DocStringExtensions to 0.9, (keep existing compat) https://github.com/QEDjl-project/QEDbase.jl/pull/40
+* add CompatHelper CI Job https://github.com/QEDjl-project/QEDbase.jl/pull/35
+* run unit tests for Julia 1.6 until 1.9
+https://github.com/QEDjl-project/QEDbase.jl/pull/29
+* add documentation build and deploy job
+https://github.com/QEDjl-project/QEDbase.jl/pull/28
+* Belated review fixes on #25
+https://github.com/QEDjl-project/QEDbase.jl/pull/34
+* remove GitLab CI formatter job
+https://github.com/QEDjl-project/QEDbase.jl/pull/27
+* add formatter job to GitHub Actions
+https://github.com/QEDjl-project/QEDbase.jl/pull/19
+* [FIX-21] building docs locally
+https://github.com/QEDjl-project/QEDbase.jl/pull/23
+* remove Mainifest.toml https://github.com/QEDjl-project/QEDbase.jl/pull/18
+* make format_all.jl path independent
+https://github.com/QEDjl-project/QEDbase.jl/pull/17
+* add integration tests https://github.com/QEDjl-project/QEDbase.jl/pull/2
+* ci: add unit tests https://github.com/QEDjl-project/QEDbase.jl/pull/13
+* Update the gitignore to fix issue #6
+https://github.com/QEDjl-project/QEDbase.jl/pull/11
```

Comment:
There are a number of irrelevant PRs listed here. We should probably create a 'changelog' marker and add it only to the PRs that are relevant for users in the end.

Reply:
You are right, of course, some of the PRs are just minor changes. However, I think we should list all PRs included in a release somewhere, e.g. in the `CHANGELOG.md`. As a compromise, I suggest having a highlight section, which could be filled with information about the user-relevant PRs.

Reply:
I am fine for now, this was mainly a suggestion for the future. I think if someone wants to see all the changes, they would just scroll through the list of closed PRs. That is what it is for, right? So I would put only 'user relevant' changes in the changelog.